app/packs/src/apps/moleculeModerator/MoleculeModeratorComponent.js

```diff
@@ -184,131 +194,163 @@ export default class MoleculeModeratorComponent extends Component {
-        <td>{i + 1}</td>
-        <td>
-          {this.renderDeleteButton(na)}
-          &nbsp;
-          {this.renderEditButton(na)}
-        </td>
-        <td>{na.description}</td>
-        <td>{na.name}</td>
-      </tr>
-    ));
-
     return (
-      <div>
+      <>
         {componentEditor}
-        <div className="container">
-          <Panel>
-            <Panel.Heading>
-              <b>InChiKey:</b>&nbsp;{this.props.molecule.inchikey}
-              &nbsp;(<b>Chemotion molecule id:</b>&nbsp;{this.props.molecule.id})
+        <Container>
+          <Card>
+            <Card.Header>
+              <b>InChiKey:</b>
+              {' '}
+              {molecule.inchikey}
+              {' '}
+              (
+              <b>Chemotion molecule id:</b>
+              {' '}
+              {molecule.id}
+              )
               <br />
-              <b>Canonical Smiles:</b>&nbsp;{this.props.molecule.cano_smiles}
-            </Panel.Heading>
-            <Panel.Body>
-              <Row>
-                <Col md={12}>
-                  <Button variant="primary" size="sm" onClick={() => this.props.handleEditor(true)}>Open Editor&nbsp;<i className="fa fa-pencil" aria-hidden="true" /></Button>&nbsp;
-                  <Button variant="warning" size="sm" onClick={() => this.handleSave()}>Update molfile and svg&nbsp;<i className="fa fa-floppy-o" aria-hidden="true" /></Button>
+              <b>Canonical Smiles:</b>
+              {' '}
+              {molecule.cano_smiles}
+            </Card.Header>
+            <Card.Body>
+              <Row className="mb-3">
+                <Col>
+                  <Button variant="primary" size="sm" onClick={() => handleEditor(true)}>
+                    Open Editor
+                    <i className="fa fa-pencil ms-1" aria-hidden="true" />
+                  </Button>
+  &nbsp;
+                  <Button variant="warning" size="sm" onClick={() => this.handleSave()}>
+                    Update molfile and svg
+                    <i className="fa fa-floppy-o ms-1" aria-hidden="true" />
+                  </Button>
                 </Col>
               </Row>
-              <Row>
-                <Col md={4}>
-                  <b>molfile:</b><br />
-                  <FormControl componentClass="textarea" placeholder="textarea" value={this.props.molecule.molfile} readOnly style={{ minHeight: 'calc(50vh)' }} />
+              <Row className="mb-3">
+                <Col xs={4}>
+                  <b>molfile:</b>
+                  <br />
+                  <Form.Control
+                    as="textarea"
+                    readOnly
+                    placeholder="textarea"
+                    value={molecule.molfile}
+                    style={{ minHeight: 'calc(50vh)' }}
+                  />
                 </Col>
-                <Col md={8}>
-                  <b>svg:</b><br />
+                <Col xs={8}>
+                  <b>svg:</b>
+                  <br />
```

Comment:
Div is a block-level element. It does not need a <br /> tag to go in it's own row. If you need extra space use spacing utility classes.